src/core/net/netif.cpp

```diff
@@ -281,6 +284,13 @@ ThreadError Netif::AddUnicastAddress(NetifUnicastAddress &aAddress)
     aAddress.mNext = mUnicastAddresses;
     mUnicastAddresses = &aAddress;
 
+    if (aAddress.GetAddress().m16[4] == HostSwap16(0x0000) &&
```

Comment:
I think it would be better to add a `Ip6::Address::IsRoutingLocator()` method that can be used here and below.  Also, let's pull `mUnicastChangedTask.Post()` into the body of this if-statement to make it consistent with the code below as well.